helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -349,6 +349,7 @@ logsCollection:
     # identify source of the logs: namespace, pod and container names. At least one source
     # identifier has to be specified in for each multiline config.
     multilineConfigs: []
+
```

Comment:
This is a wrong place to separate the comments :)

Comments below are still applied to multiline config and mixed with comments for `useSplunkIncludeAnnotation`